.github/workflows/images.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          platforms: linux/amd64,linux/arm64
```

Comment:
@nerdalert just to make sure, do you want to skip building gobot image for linux/arm64 or for worker image ?

Reply:
Roger, it's broken and takes 35 minutes just to fail. Backend is asking for an image so we need to get this out. Right now the newly named GHCR has no images. Ty

Reply:
Good Catch, wrong job. <3